src/main/java/core/basesyntax/model/controller/ConsoleHandler.java

```diff
@@ -0,0 +1,44 @@
+package core.basesyntax.model.controller;
+
+import core.basesyntax.model.dao.BetDaoImpl;
+import core.basesyntax.model.dao.Dao;
+import core.basesyntax.model.dao.UserDaoImpl;
+import core.basesyntax.model.model.Bet;
+import core.basesyntax.model.model.User;
+import java.util.Scanner;
+
+public class ConsoleHandler {
+    private Dao betDao = new BetDaoImpl();
+    private Dao userDao = new UserDaoImpl();
```

Comment:
you can use one generic interface for all dao, but you still should have its own interface so you(they may extend Generic) should be able to create BetDao betDao = new BetDaoImpl()